scheduler/src/cook/mesos.clj

```diff
@@ -178,6 +178,7 @@
         datomic-report-chan (async/chan (async/sliding-buffer 4096))
         mesos-heartbeat-chan (async/chan (async/buffer 4096))
         current-driver (atom nil)
+        pool-name->optimizer-schedule-job-ids-atom (atom {})
```

Comment:
This name is very confusing. We should try to come up with something simpler.
It looks like it's a mapping from pool names onto atoms.
Are the mapped-to elements schedules, or just a collection of job ids, or something else?